packages/babel-helpers/src/helpers/applyDecs.js

```diff
@@ -207,9 +209,9 @@ function memberDecCtx(
   );
 }
 
-function assertValidInitializer(initializer) {
-  if (typeof initializer !== "function") {
-    throw new Error("initializers must be functions");
+function assertCallable(fn, hint) {
+  if (typeof fn !== "function") {
+    throw new TypeError(hint + " must be functions");
```

Comment:
```suggestion
    throw new TypeError(hint + " must be a function");
```
since `hint` values are singular.

Reply:
:+1: for `An initializer must be a function`